docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx

```diff
@@ -16,6 +16,13 @@ In this guide, we'll walk you through creating and revoking user and agent token
   <strong>Editor</strong>, <strong>Admin</strong>, or{" "}
   <strong>Organization Admin</strong> permissions are required to manage agent
   tokens. This includes viewing existing tokens.
+  <br />
+  <br />
+  If you're a Dagster Cloud <a href="/dagster-cloud/account/managing-users">
+    <strong>Editor</strong> or <strong>Admin</strong>
+  </a>, you can only manage agent tokens in deployments where you're an <strong>
+    Editor
+  </strong> or <strong>Admin</strong>.
```

Comment:
discussed offline to (aspirationally - i'll make it true shortly) change this to 'editor or admin on at least one deployment (or org admin)'